src/paths/build_output_dir.py

```diff
@@ -9,4 +9,4 @@
 
 class BuildOutputDir(OutputDir):
     def __init__(cls, name, cwd=None, makedirs=True):
-        super().__init__("builds", name, cwd, makedirs)
+        super().__init__("builds", name.lower().replace(" ", "-"), cwd, makedirs)
```

Comment:
Did you also mean to change this?

Reply:
No. This is from InputManifest https://github.com/opensearch-project/opensearch-build/blob/main/src/run_build.py#L27 which do not have the filename property.

Reply:
Should we add it?

Reply:
Added. Now the logic for build and assemble look very unified